public/room.js

```diff
@@ -172,18 +171,30 @@ function socketHandlers(socket) {
         addTextToNote(noteEventMessage['noteId'], noteEventMessage['text']);
         addNoteToLocalStorage(noteEventMessage['noteId'], noteEventMessage['text']);
         break;
+      case 'nodeDeleted':
+        if (noteEventMessage['noteId'] == 1) {
+            addNoteToLocalStorage(1, 'This is a sticky note you can type and edit.');
+            $('.sticky-note[data-note-id="1"]').html('This is a sticky note you can type and edit.');
+            break;
+        }
+        addNoteToLocalStorage(noteEventMessage['noteId'], null);
```

Comment:
This should not add null, but remove that key. Add new method `removeNoteFromLocalStorage` for the same